config/locales/en.yml

```diff
@@ -23,7 +23,7 @@ en:
       update: :thredded.form.update
       updated_notice: Messageboard has been updated
     messageboard_group:
-      create: Create a New Messageboard Group
+      create: Messageboard group has been created
```

Comment:
This key is used in a few other places where this text would not make sense:

```
$ i18n-tasks find thredded.messageboard_group.create 
1 key matching 'thredded.messageboard_group.create' (4 usages) | i18n-tasks v0.9.5
thredded.messageboard_group.create 4
  app/views/thredded/messageboard_groups/new.html.erb:1 <% content_for :thredded_page_title, t('thredded.messageboard_group.create') %>
  app/views/thredded/messageboard_groups/new.html.erb:5 <li><%= t('thredded.messageboard_group.create') %></li>
  app/views/thredded/messageboard_groups/new.html.erb:17 <li><%= f.submit t('thredded.messageboard_group.create'), class: 'thredded--form--submit' %></li>
  app/views/thredded/messageboards/index.html.erb:18 <a class="thredded--button" href="<%= new_messageboard_group_path %>"><%= t('thredded.messageboard_group.create') %></a>
```

Please add a new `created_notice` key instead.